doc/whats-new.rst

```diff
@@ -55,6 +55,10 @@ Enhancements
   (:issue:`576`).
   By `Stephan Hoyer <https://github.com/shoyer>`_.
 
+- Deprecate supports of old numpy < 1.11 and pandas < 0.18 (:issue:`1512`).
```

Comment:
This should go to the top of the release notes, under a heading "Backwards incompatible changes"

Also: these versions aren't just deprecated -- they are no longer supported at all.